airflow/contrib/operators/file_to_gcs.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from airflow.contrib.hooks.gcs_hook import GoogleCloudStorageHook
+from airflow.models import BaseOperator
+import os
+
+
+class FileToGoogleCloudStorageOperator(BaseOperator):
+    """
+    Uploads a file to Google Cloud Storage
+    """
+
+    def __init__(self,
+                 src,
+                 dst,
+                 bucket,
+                 google_cloud_storage_conn_id='google_cloud_storage_default',
+                 delegate_to=None,
+                 *args,
+                 **kwargs):
+        """
+        :param src: Path to the local file
+        :type src: string
+        :param dst: Destination path within the specified bucket
+        :type dst: string
+        :param bucket: The bucket to upload to
+        :type bucket: string
+        :param google_cloud_storage_conn_id: The Airflow connection ID to upload with
+        :type google_cloud_storage_conn_id: string
+        :param delegate_to: The account to impersonate, if any
+        :type delegate_to: string
+        """
+        super(FileToGoogleCloudStorageOperator, self).__init__(*args, **kwargs)
+        self.src = src
+        self.dst = dst
+        self.bucket = bucket
+        self.google_cloud_storage_conn_id = google_cloud_storage_conn_id
+        self.delegate_to = delegate_to
+
+    def execute(self, context):
+        """
+        Uploads the file to Google cloud storage
+        """
+        hook = GoogleCloudStorageHook(
+                google_cloud_storage_conn_id=self.google_cloud_storage_conn_id,
+                scope='https://www.googleapis.com/auth/devstorage.read_write',
```

Comment:
No need for scope anymore. This is being handled at the connection level in the latest version of airflow.